include/api.php

```diff
@@ -114,7 +114,7 @@ function pll_default_language( $field = 'slug' ) {
  *
  * @param int                 $post_id Post ID.
  * @param PLL_Language|string $lang    Optional language (object or slug), defaults to the current language.
- * @return int|false The translation post ID if exists, otherwise the passed ID. False if the passed object has no language or if the language doesn't exist.
+ * @return int|false The translation post ID if exists, 0 if not translated. False if the passed object has no language or if the language doesn't exist.
```

Comment:
Shouldn't it be : 
```suggestion
 * @return int|false The translation post ID if exists, 0 if not translated or if the passed object has no language. False if the language doesn't exist.
```

Reply:
Indeed, see d192b43a. I also improve the returned value when the language doesn't exist (according to the documentation).